packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/index.tsx

```diff
@@ -21,7 +21,7 @@ export type ConfirmResetPasswordLocationState = {
   passwordForgotToken: string;
 };
 
-const POLLING_INTERVAL_MS = 2000;
+const POLLING_INTERVAL_MS = 4000;
```

Comment:
Was this for debugging, or did we mean to change a polling interval from 2s to 4s?

Reply:
This was from some testing, but the default `VERIFICATION_POLL_IN_MS` from content-server was 4000. I'm not sure if it matters or not if we match that. @vbudhram was the 2000ms arbitrary, or is there are reason to change? I'll set up a constant and use the same value for both `ConfirmResetPassword` and `ResetPassword`.

Reply:
I don't think it matters, the 2second I thought gave a slightly better user experience than 4 seconds. Your call :)